packages/graphql-transformers-e2e-tests/src/__tests__/FunctionTransformerTestsV2.e2e.test.ts

```diff
@@ -39,9 +46,98 @@ function outputValueSelector(key: string) {
     const output = outputs.find((o: Output) => o.OutputKey === key);
     return output ? output.OutputValue : null;
   };
-}
+};
+
+const createEchoFunctionInOtherAccount = async (currentAccountId?: string) => {
+  if (!currentAccountId) {
+    return;
+  }
+  try {
+    const childAccounts = (await organizations.listAccounts({}).promise())?.Accounts;
+    if (!childAccounts || childAccounts?.length < 1) {
+      console.warn('Could not find any child accounts attached to current account');
+      expect(true).toEqual(false);
+      return;
+    }
+    const otherAccountId = childAccounts[0]?.Id;
+    if (!otherAccountId) {
+      console.warn('Could not choose other account to create lambda function');
+      expect(true).toEqual(false);
+      return;
+    }
+    const childAccountRoleARN = `arn:aws:iam::${otherAccountId}:role/OrganizationAccountAccessRole`;
+    const accountCredentials = (await sts.assumeRole({ 
+      RoleArn: childAccountRoleARN, 
+      RoleSessionName: `testCrossAccountFunction${BUILD_TIMESTAMP}`,
+      DurationSeconds: 900
+    }).promise())?.Credentials;
+    if (!accountCredentials?.AccessKeyId || !accountCredentials?.SecretAccessKey || !accountCredentials!.SessionToken) {
+      console.warn('Could not assume role to access child account');
+      expect(true).toEqual(false);
+      return;
+    }
+    const crossAccountLambdaHelper = new LambdaHelper(region, new AWS.Credentials( accountCredentials.AccessKeyId, accountCredentials.SecretAccessKey, accountCredentials.SessionToken));
+    const crossAccountIAMHelper = new IAMHelper(region, new AWS.Credentials( accountCredentials.AccessKeyId, accountCredentials.SecretAccessKey, accountCredentials.SessionToken));
+    const role = await crossAccountIAMHelper.createLambdaExecutionRole(LAMBDA_EXECUTION_ROLE_NAME);
+    await wait(5000);
```

Comment:
Is this `wait` used for lambda role to be processed after the creation? If so we can add it as a constant at the beginning of the code like `LAMBDA_CREATION_DELAY = 5000`

Reply:
I will make these as constants, thanks.